# An array of poses with a header for global reference.

Header header

localization/PoseWithWeight[] poses
